### PR TITLE
Artifact fix templates

### PR DESCRIPTION
**Pull request by josenavas**
Depends on https://github.com/biocore/qiita/pull/1535

Fixes the metadata template tests.

### CODE REVIEW

**Comment by antgonza** [comment]
:+1:

**Comment by mortonjt** [comment]
Looks like we are having some weird import errors here.

**Comment by antgonza** [comment]
The failures are expected as the modifications to make everything pass are large == difficult to review.

File: qiita_db/metadata_template/prep_template.py [hunk 1]
```diff
@@ -361,15 +354,17 @@ def can_be_extended(self, new_samples, new_columns):
...
-                                   "template")
+                artifact = self.artifact
+                if artifact:
+                    sql = """SELECT EXISTS(SELECT *
+                                           FROM qiita.parent_artifact
+                                           WHERE parent_id = %s)"""
+                    qdb.sql_connection.TRN.add(sql, [artifact.id])
+                    if qdb.sql_connection.TRN.execute_fetchlast():
+                        return False, ("The artifact attached to the prep "
+                                       "template has been already processed. "
```
Comment (ElDeveloper) [inline qiita_db/metadata_template/prep_template.py]: been already -> already been

**Comment by josenavas** [comment]
@mortonjt that is expected - note that this is going to its own branch and I have to break the system in order to do all the needed changes in a 'reviewable' manner - i.e. if I fix everything - the changes will be so big that github will not show the changes done so far.

This should fix the tests for the qiita_db subpackage, while qiita_ware and qiita_pet are not fixed given that they need to change.

**Comment by ElDeveloper** [comment]
:+1:

**Comment by mortonjt** [comment]
Alrighty, that makes sense.

**Pull request merged**